tests/functional/network_policy/conftest.py

```
@@ -0,0 +1,119 @@
+                        "runAsNonRoot": True,
+                        "seccompProfile": {"type": "RuntimeDefault"},
+                    },
+                }
+            ],
+        },
+    }
+
+    import tempfile
+    from ocs_ci.utility.templating import dump_data_to_temp_yaml
+
+    tmp = tempfile.NamedTemporaryFile(
```

Ruff (0.16.0)

[warning] Use a context manager for opening files

(SIM115)